cmd/package_managers.go

```diff
@@ -138,3 +202,115 @@ func fetchGitRepositoryFromRubyGems(packageName string, manager packageManagerCl
 	}
 	return v.SourceCodeURI, nil
 }
+
+// Gets the GitHub repository URL for the nuget package.
+func fetchGitRepositoryFromNuget(packageName string, manager packageManagerClient) (string, error) {
```

Comment:
Is `packageName` guaranteed to be lowercase at this point? The URLs always contain lowercase ID and version strings for the V3 API.

Reply:
Lowercase is done via .NET's `ToLowerInvariant()`. It might be a nightmare trying to get the same lowercase behavior in Go. I've haven't studies lowercase folding behavior in Go at all but it's surprisingly nuanced in .NET.

Reply:
Seeing as the functionality impacts only users using `--nuget` I'm fine with taking the simple approach for now, and someone filing an issue later if it's an issue.